src/recipe/dto/fetch-recipes-response.ts

```diff
@@ -0,0 +1,9 @@
+import { Recipe } from '@prisma/client';
+
+export class FetchRecipesResponse {
+  constructor(public fetchedRecipes: Recipe[]) {}
+
+  public static from(fetchedRecipes: Recipe[]): FetchRecipesResponse  {
```

Comment:
double space - run prettier
```suggestion
  public static from(fetchedRecipes: Recipe[]): FetchRecipesResponse {
```